crates/bevy_animation/src/lib.rs

```diff
@@ -46,6 +46,22 @@ pub enum Keyframes {
     Weights(Vec<f32>),
 }
 
+impl Keyframes {
+    /// Returns the number of keyframes.
+    pub fn len(&self) -> usize {
+        match self {
+            Keyframes::Weights(vec) => vec.len(),
+            Keyframes::Translation(vec) | Keyframes::Scale(vec) => vec.len(),
```

Comment:
Why are these two on the same line while the others are split up?
```suggestion
            Keyframes::Translation(vec) => vec.len(),
            Keyframes::Scale(vec) => vec.len(),
```

Reply:
because `vec` is the same type for translation and scale, but not for weights

Reply:
Yep I would do all 4 on one line but it didn't compile.